### PR TITLE
Fix: version

### LINKED ISSUE #288: --version is broken
🙀

### PR DESCRIPTION
**Pull request by stramel**
@FredKSchott Fixes #288 

Also, ran format.

### CODE REVIEW

**Comment by stramel** [comment]
> By default, `--max_old_space_size` (which controls the upper limit of the V8 heap) is ~1.5GB. If your system has less memory than that, you will need to explicitly set this option to the appropriate amount of memory available to that process.

Maybe it would be better to not have this on there by default and just add a note in the README?

**Comment by FredKSchott** [comment]
That's a fair point, we set that as the default because we were running into it for the 100+ element runs we've been doing.

If your machine has less memory than 8192MB, what happens? I assumed it still ran and would only cause issues if you were only trying to use more memory than your machine had available, which would be the case in either case.

**Comment by FredKSchott** [comment]
From @stramel in Slack:
> Yeah, so with the `--max-old-space-size=8192` it just spins out and uses my system's resources
> once removed i get the `../package.json` error

Okay then, I think that's safe to remove now. We're getting ready to bring our own process into a single executable file / shell script, so we can handle this there for our specific need instead

File: package.json [hunk 1]
```diff
@@ -19,7 +19,7 @@
     "test:watch": "watchy -w src/ -w custom_typings/ -- npm run test:fast --loglevel=silent",
     "test": "npm run test:fast && npm run lint",
     "lint": "tslint -p ./",
-    "format": "find src test | grep '\\.js$\\|\\.ts$' | xargs clang-format --style=file -i",
+    "format": "find bin src test | grep \"\\.js$\\|\\.ts$\" | xargs clang-format --style=file -i",
```
Comment (stramel) [inline package.json]: This command will properly run on windows now too.

Comment (FredKSchott) [reply package.json]: nice!

**Comment by FredKSchott** [comment]
Thanks @stramel!

**Pull request merged**